frontend/src/app/urlToCitation.py

```diff
@@ -0,0 +1,76 @@
+import requests
+from bs4 import BeautifulSoup
+from datetime import date
+
+# Takes list of string urls
+def urlToCitation(url):
```

Comment:
Maybe consider putting this file inside another folder under app, maybe something like `services`